src/MembershipsSDKAdapter.js

```diff
@@ -51,42 +51,33 @@ export default class MembershipsSDKAdapter extends MembershipsAdapter {
       return throwError(new Error(`getMembersFromDestination for ${destinationType} is not currently supported.`));
     }
 
+    const membershipID = `${destinationType}-${destinationID}`;
+
+    if (this.membershipSubjects[membershipID]) {
+      return this.membershipSubjects[membershipID];
+    }
+
     const meeting = this.datasource.meetings.getMeetingByType('id', destinationID);
 
     if (!meeting) {
       return throwError(new Error(`Meeting ${destinationID} not found.`));
     }
 
-    const membershipID = `${destinationType}-${destinationID}`;
+    const members = meeting.members
```

Comment:
Maybe rename this to make it a little bit clearer that is meant as a boolean?
```suggestion
    const hasMembers = meeting.members
```

Reply:
it's actually the member list from the sdk, but "prudently" extracted from the inner data structure of member collection.

If we had the optional chaining operator, we could write a more concise:
```js
const members = meeting.members?.membersCollection?.members;
```

This replaces the `if` on line 75 of the previous version of this file.

If you prefer, I can move the check inside the `getActiveMembers` function, so that it returns an empty array if it receives a nullish `members` argument. As a benefit of this approach, a `members:update` event with an empty `payload.full` property would no longer generate an error:

```js
const updatedMembers = getActiveMembers(payload.full);
```

Reply:
I like your suggestion on refactoring `getActiveMembers`, it's cleaner 👍🏼

Reply:
thanks!

Reply:
@lalli-flores I rebased this PR onto the (fixed) master branch and implemented the null check in `get(Active)Members`